build.gradle

```diff
@@ -35,6 +35,18 @@ configurations.all {
     resolutionStrategy {
         force "com.google.appengine:appengine-api-1.0-sdk:${appengineVersion}"
         force "org.jsoup:jsoup:${jsoupVersion}"
+        eachDependency { DependencyResolveDetails details ->
+            // Replace all instances of Guava JDK5 Backport with vanilla Guava.
+            // We have to be very careful when updating libraries that depend on Guava and make sure
+            // the Guava version used in every library is not too far away from the Guava JDK5 Backport
+            // because Guava is not fully backwards compatible due to its deprecation policy.
+            // Currently, the only instance of `guava-jdk5` is used by `google-api-java-client`.
+            // We can monitor https://github.com/google/google-api-java-client/issues/903
```

Comment:
This issue has been closed. I think the more correct description will be when a new version of the library (likely `1.24.0`) is released?